spec/std/math_spec.cr

```diff
@@ -41,6 +41,20 @@ describe "Math" do
       Math.sqrt(4_f32).should eq(2)
       Math.sqrt(4).should eq(2)
     end
+
+    it "isqrt" do
+      Math.isqrt(9).should eq(3)
+      Math.isqrt(8).should eq(2)
+      Math.isqrt(4).should eq(2)
+      Math.isqrt(1764).should eq(42)
+      {% for type in [UInt8, UInt16, UInt32, Int8, Int16, Int32, Int64] %}
+        %val = {{type}}::MAX
+        %exp = {{type}}.new Math.sqrt(%val)
+        Math.isqrt(%val).should eq(%exp)
+      {% end %}
+      # Floating point implementation looses precision at UInt64 sizes.
+      Math.isqrt(UInt64::MAX).should eq(4294967295)
```

Comment:
Could you reword the comment, it's hard to understand. Without context I would assume it describes the next expectation as inaccurate, but representing inaccurate implementation.
But the implementation doesn't use floating point arithmetics, so it's actually the opposite, right?

Reply:
Can do. That was a note on why this test case is different to the others above. Will make that a more clear.